README.md

```diff
@@ -205,6 +208,55 @@ gh api \
 jq '[.[] | {login}]'
 ```
 
+This generates the file from all users in a GitHub organization. The same
+concept could also be used for a GitHub team. Of course, the JSON file can be
+generated by any means, as long as it follow the expected format.
+
+##### Inputs
+
+One of `team`, `local-members-path`, or `members-url` must be set. If multiple
+of these inputs are set, only one will be used. Which one will be used is
+undefined, but deterministic.
+
+**team**: This is actually the org and team formateted as ${org}/${team-slug}
+e.g. abcxyz/seahawks-2013.
+
+**local-members-path**: JSON file that exists in the repository that is calling
+the workflow.
+
+**members-url**: Full URL to a members JSON file. If this is in a public GitHub
+repository, the URL should be similar to
+https://raw.githubusercontent.com/${owner}/${repo}/${ref}/path/to/members.json.
+
+##### Teams
+
+If you use the `team` input, you will have to setup
+[github-token-minter](https://github.com/abcxyz/github-token-minter)  to provide
+a token with elevated priviledges (necessary to query a user's membership in a
+team). The github-token-minter scope configuration should be as follows:
+
+```yaml
+scope:
+  your-name: # Set this to whatever you want.
+    rule:
+      if: |-
+        # Check that the request is coming from the expected repository.
+        # The ID should be replaced with yours.
+        assertion.repository_id == '12345678' &&
```

Comment:
Can we use repository name, such as "google-cloud-java" for https://github.com/googleapis/google-cloud-java? (I've never referenced a GitHub repository with a number. Is there a benefit in this?)

Reply:
Repository names are mutable **and** subject to name squatting attacks. Using the repository id is the correct way to ensure the request is coming from the repository that it is expected to come from.

Reply:
Great. That's worth mentioning here.